### DISCUSSION TITLE
Join Addresses not showing up?

### DISCUSSION THREAD

**Discussion by Tosfoy** [plan-player-analytics/Plan#2807] [APIs and Development]
This probably does not belong here, but I haven't detected any bugs with plan, but this is related to the plugin.
Our Join Addresses says "No Data to Display"
![image](https://user-images.githubusercontent.com/58991099/209907756-99c316df-cb0c-48f1-93ae-55ed5f43f2d7.png)

So I am unsure if this happened after upgrading to our new machine and uploading the databases, which I doubt
or
This is because of TCPShield, as we now pay for their plan which offers analytics for Top Domains, problem is they don't show as much as Plan does.
![image](https://user-images.githubusercontent.com/58991099/209907768-5c486d87-7cb8-4184-9272-ceeda2818b78.png)

Do you think this could be an issue with TCPShield, or Plan itself?
Let me know!

**Comment by AuroraLS3** [comment]
Please check if `plan_join_addresses` table has data in your database
